docs/source/60_building_apps.md

```diff
@@ -256,6 +256,39 @@ $ spark-submit \
 A complete example of this pattern can be found in the
 [heuermh/adam-examples](https://github.com/heuermh/adam-examples) repository.
 
+### Using ADAM’s ShuffleRegionJoin API
+Another useful API implemented in ADAM is the ShuffleRegionJoin API, which joins two genomic datasets that contain overlapping regions. This primitive is useful for a number of applications including Variant calls and Feature mapping.
```

Comment:
3. This section should talk about the distinction between a shuffle and broadcast region join, and why one is preferable to the other.